language/move-lang/tests/move_check/unit_test/script_with_multiple_on_main.exp

```diff
@@ -0,0 +1,19 @@
+error: 
+
+   ┌── tests/move_check/unit_test/script_with_multiple_on_main.move:4:7 ───
+   │
+ 4 │     #[expected_failure]
+   │       ^^^^^^^^^^^^^^^^ Testing attributes are not allowed in scripts.
+   │
+
+error: 
+
+   ┌── tests/move_check/unit_test/script_with_multiple_on_main.move:5:7 ───
+   │
+ 5 │     #[test_only, test]
+   │       ^^^^^^^^^ Testing attributes are not allowed in scripts.
+   ·
+ 5 │     #[test_only, test]
+   │                  ---- Testing attributes are not allowed in scripts.
+   │
+
```

Comment:
Why aren't these separate errors?

Reply:
I think it's because they're in the same annotation. I'll split them out.